### ISSUE TITLE
Example with different volumes

### ISSUE THREAD

**Issue by rogerkuou** [eEcoLiDAR/tutorial_ecolidar#2]
We can add examples with different volumes. 
 E.g. we can show examples of computing eigenvectors to check if a point belongs to a certain surface, as it was mentioned in the paper: https://www.sciencedirect.com/science/article/pii/S2352711020303393

**Comment by meiertgrootes** [comment]
Good idea. We can lift the code for that from the paper code. I'll check where it could be
